src/web/command_control/client/style/CommandControl.less

```diff
@@ -1349,10 +1348,10 @@ div.closeButton {
   margin: 4pt 8pt 0 8pt;
   padding: 2pt;
   
-  background-color: #FFFFFF;
-
+  color: #FFFFFF;
```

Comment:
Should we use a variable here for the color? Util.less is probably the spot for this

Reply:
Just added a variable...there is a little overlap with the variable names, but I think cc-text-color-light (I would prefer jcc-text-color-light but that is another ticket) is more reusable than the variable names specific for the panels (for instance there is @cc-accordian-text-color: white; but I find that a little confusing because it gets into the definition of an accordion (is it just the gray bar or does it include the drop down too which is black text)